app/controllers/admin/answers_controller.rb

```diff
@@ -32,6 +32,7 @@ def set_question
     @question = Question.find(params[:question_id])
   end
 
+  
```

Comment:
Can you remove this? Small stuff, but if it's not necessary it shouldn't be here.

Reply:
It should be removed now.